packages/dds/test-dds-utils/src/fuzzSerializer.ts

```diff
@@ -0,0 +1,223 @@
+/*!
```

Comment:
this is copied right now from existing serializer + then some changes applied. I'm not really a fan of this. FluidSerializer is actually internal so it shouldn't be too bad to refactor it some so that I can just inject the 'poisoning' logic. Open to thoughts from the reviewer though.

Reply:
maybe split the difference and just use inheritance. it will make it clearer which code is really needed vs not.